### DISCUSSION TITLE
Adding in a Second Inverter

### DISCUSSION THREAD

**Discussion by gjhill6100** [nathanmarlor/foxess_modbus#262] [Q&A] [answered]
When you come to add a second inverter to the system, when using the Integration, do you just add in the second Inverter as a new single Inverter in the configuration or re-run the configuration with the original inverter with the same IP address and local name and then the second Inverter with the new IP and new second inverter name ?

**Comment by canton7** [comment]
The configuration won't let you add a second inverter later. Delete the integration (3 dots in the bottom right -> delete), then add it again, adding both inverters. As long as the entity ID prefix of the existing inverter doesn't change, none of your entities will be lost.

**Reply by canton7** [reply]
(https://github.com/nathanmarlor/foxess_modbus/wiki/FAQs#how-do-i-change-my-inverter-settings-)

**Reply by gjhill6100** [reply, marked as answer]
Many thanks for that. I will give that a go.
